torch/_inductor/codecache.py

```diff
@@ -1303,6 +1533,12 @@ def task():
 
         return self.submit(task)
 
+    def cuda(self, dst_code_suffix, source_code):
```

Comment:
I'd like to keep the order here so that it's easier to review generated code. Otherwise the source code part is too large and will make the other params non-visible.